react/typings/vtex.styleguide.d.ts

```diff
@@ -1,8 +1,6 @@
 /* tslint:disable:max-classes-per-file class-name*/
 declare module 'vtex.styleguide' {
   import { Component, ComponentType, ReactNode } from 'react'
-  // Remove this dependency from admin-pages when styleguide is properly typed.
-  import { ReactDatePickerProps } from 'react-datepicker'
```

Comment:
Hm, I know this wasn't the best solution, but at the same time, we just lost static typing on the `DatePicker` by removing this :S

Reply:
Good point!
I've added the [types from DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c68ed87e3fd6a1433c4b28aa11c12b27af25d948/types/react-datepicker/index.d.ts) as a dev dependency, and then put `ReactDatePickerProps` back in the `DatePickerProps`.